unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/AvatarShape/Tests/AvatarShapeComponentHandlerShould.cs

```diff
@@ -0,0 +1,163 @@
+﻿using DCL.Controllers;
+using DCL.Models;
+using NSubstitute;
+using NSubstitute.Extensions;
+using NSubstitute.ReceivedExtensions;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace DCL.ECSComponents.Tests
+{
+    public class AvatarShapeComponentHandlerShould
+    {
+        private const string AVATAR_POOL_NAME = "AvatarShapeECS";
+        
+        private IDCLEntity entity;
+        private IParcelScene scene;
+        private IAvatarShape avatarShape;
+        private AvatarShapeComponentHandler componentHandler;
+        private GameObject gameObject;
+        
+        private Pool pool;
+
+        [SetUp]
+        protected void SetUp()
+        {
+            gameObject = new GameObject();
+            entity = Substitute.For<IDCLEntity>();
+            scene = Substitute.For<IParcelScene>();
+            AvatarShape avatarShapePrefab = Resources.Load<AvatarShape>("NewAvatarShape");
+            ConfigurePool(avatarShapePrefab.gameObject);
+            componentHandler = new AvatarShapeComponentHandler(pool);
```

Comment:
There must be a better way to mock this, Handler tests should not need the real prefab of the AvatarShape whatsoever.